webapp/src/components/shareBoard/teamPermissionsRow.tsx

```diff
@@ -91,13 +91,15 @@ const TeamPermissionsRow = (): JSX.Element => {
                                     name={intl.formatMessage({id: 'BoardMember.schemeCommenter', defaultMessage: 'Commenter'})}
                                     onClick={() => updateBoardType(board, BoardTypeOpen, 'commenter')}
                                 />}
-                            <Menu.Text
-                                id='Viewer'
-                                check={board.minimumRole === 'viewer'}
-                                icon={board.type === BoardTypeOpen && board.minimumRole === 'viewer' ? <CheckIcon/> : null}
-                                name={intl.formatMessage({id: 'BoardMember.schemeViwer', defaultMessage: 'Viewer'})}
-                                onClick={() => updateBoardType(board, BoardTypeOpen, 'viewer')}
-                            />
+                            {board.isTemplate &&
```

Comment:
We don't need this conditional—it's why the snapshot tests are failing. It only needed to be fixed in `release-7.2` as I did earlier in #3681.

Reply:
Yep...bad merge. This change shouldn't be here.